compliance/nvidia/TEST05/run_verification.py

```diff
@@ -58,19 +58,22 @@ def main():
     verify_performance_command = "python3 " + verify_performance_binary + " -r " + results_dir + "/performance/run_1/mlperf_log_summary.txt" + " -t " + compliance_dir + "/mlperf_log_summary.txt | tee verify_performance.txt"
     try:
         os.system(verify_performance_command)
-    except:
+    exept Exception:
         print("Exception occurred trying to execute:\n  " + verify_performance_command)
 
     # check if verify performance script passes
     performance_pass_command = "grep PASS verify_performance.txt"
-    performance_pass = "TEST PASS" in subprocess.check_output(performance_pass_command, shell=True).decode("utf-8")
+    try:
+        performance_pass = "TEST PASS" in subprocess.check_output(performance_pass_command, shell=True).decode("utf-8")
+    exept Exception:
```

Comment:
except

Reply:
I can change it back but lgtm-com will flag it.

Reply:
I guess I shouldn't trust lgtm